conans/client/installer.py

```diff
@@ -319,6 +319,8 @@ def _handle_node_cache(self, node, package_id, keep_build, processed_package_ref
                         self._registry.set_ref(conan_ref, node.binary_remote.name)
                 elif node.binary == BINARY_CACHE:
                     output.success('Already installed!')
+                    _handle_system_requirements(conan_file, package_ref, self._client_cache,
```

Comment:
This shouldn't be done here. System requirements are installed:
- On download of packages
- On build

Sucessive installations shouldn't install them again, and it can make installing the deps graph very slow for already cached deps.

Reply:
Well, it won't install them again if the system reqs folder is already there. But it nicely covers the case where the user wants to force the install by removing the folder.

Reply:
It seems a very corner case for me, to cover the case where someone enters the conan cache and removes there manually some folder. The problem I see is that this could increase an order of magnitude the otherwise no-op "conan install" command when the dependencies are already installed in the cache.

Reply:
I could agree with that. If it causes some minimal issue then we should avoid it. Then we should go to the issue, to comment that this is not a valid possibility and close it.